erp/management/commands/start_scheduler.py

```diff
@@ -23,6 +23,7 @@ def handle(self, *args, **options):
         schedule.every(3).hours.do(call_command, "notify_changed_erps", hours=3)
         schedule.every().hour.do(call_command, "import_dataset", "vaccination")
         schedule.every().day.do(call_command, "import_dataset", "gendarmerie")
+        schedule.every().day.at("9:30").do(call_command, "notify_unpublished_erps")
```

Comment:
Should we send a reminder every day? We're going to be flagged as spam rather quickly… How about once a week? or even once a month?

Reply:
That's why I have a _due_time() guard. The user is notified only once per week (day diff mod 7) after the last updated_at, if the ERP is not published.

Reply:
Ah yes indeed I reviewed to fast :D

Reply:
Every 7 days to be precise